concent_api/core/views.py

```diff
@@ -586,3 +719,20 @@ def get_file_status(file_transfer_token_from_database: message.concents.FileTran
         return False
     else:
         raise exceptions.UnexpectedResponse()
+
+
+def decode_key(key: str) -> bytes:
+    return b64decode(key.encode('ascii'), validate = True)
```

Comment:
This could be moved to `utils`. We're decoding keys in a lot of places, e.g. in gatekeeper. Would be convenient to have it as a shortcut.

Reply:
:+1: